frontend/libs/studio-components/src/components/StudioGridSelector/StudioGridSelector.test.tsx

```diff
@@ -0,0 +1,58 @@
+import React from 'react';
+import { fireEvent, render, screen } from '@testing-library/react';
+import { StudioGridSelector } from './StudioGridSelector';
+
+describe('StudioGridSelector', () => {
+  it('should render slider with value 12 and not disabled by default', () => {
+    render(<StudioGridSelector handleSliderChange={() => jest.fn()} />);
+
+    const slider = screen.getByRole('slider');
+    expect(slider).toHaveValue('12');
+    expect(slider).not.toHaveAttribute('disabled');
+  });
+
+  it('should render slider as disabled when disabled is true', () => {
+    render(<StudioGridSelector disabled={true} handleSliderChange={() => jest.fn()} />);
+
+    const slider = screen.getByRole('slider');
+    expect(slider).toHaveAttribute('disabled');
```

Comment:
Rather than verifying the rendering of the slider with the disabled attribute, it would be more comprehensive to test that modifying values is not possible when the slider is in a disabled state. 😊

Reply:
Even better:
```suggestion
    expect(slider).toBeDisabled();
```